components/dashboard/src/components/podkit/breadcrumbs/BreadcrumbNav.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { LinkButton } from "@podkit/buttons/LinkButton";
+import { cn } from "@podkit/lib/cn";
+import { ChevronLeft } from "lucide-react";
+import type { FC } from "react";
+
+interface BreadcrumbPageNavProps {
+    /**
+     * The title of the current page.
+     */
+    pageTitle: string;
+    /**
+     * The description of the current page.
+     */
+    pageDescription?: string;
+    /**
+     * The link to the previous page.
+     */
+    backLink?: string;
+    className?: string;
+}
+
+export const BreadcrumbNav: FC<BreadcrumbPageNavProps> = ({ pageTitle, pageDescription, backLink, className }) => {
+    return (
+        <section className={cn("flex flex-row items-center justify-start gap-2 w-full py-4 app-container", className)}>
+            {backLink && (
+                <LinkButton
+                    variant={"ghost"}
+                    className="py-1 px-0 hover:bg-gray-200 dark:hover:bg-gray-800 dark:hover:text-gray-200"
```

Comment:
We ought to have a more bare-bones `ghost` variant, or introduce the `icon` variant for easy Icon buttons.

Reply:
The fixes to ghosts are actually done in https://github.com/gitpod-io/gitpod/pull/19039. Nice!